testing/src/main/java/org/oppia/android/testing/VerifyItemDisplayedInRecyclerView.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.android.testing
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import org.oppia.android.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
```

Comment:
@anandwana001 I am facing trouble while importing this. Please guide me on what changes should I make in `build.gradle` file so that the `RecyclerViewMatcher`  could be imported. I tried adding the `app` module to the `testing` module, but that doesn't seem to work.

Reply:
I guess we can shift `RecyclerViewMatcher` to the testing module. 
Waiting for @rt4914 confirmation.

Reply:
Okay will do that after the confirmation.

Reply:
I talked to him, yes we are good to shift it.

Reply:
Shall I update the overall codebase to use this new central utility of RecyclerViewTest or shall I only update the `AdministratorControlsActivityTest` ?

Reply:
I think it will require updating all the files which are using `RecyclerViewMatcher`, as the import path will going to change for all.

Check all the files using it and try to have a successful Gradle build.